packages/badge/tests/__snapshots__/badgeButton.test.tsx.snap

```diff
@@ -82,7 +82,7 @@ exports[`BadgeButton success 1`] = `
 
 exports[`BadgeButton warning 1`] = `
 <span
-  className="css-thsdwi"
+  className="css-1i0swvi"
```

Comment:
I ran into this recently. We may reconsider snapshots if this is going to happen over and over